ztp/ran-site-plan-crd/site-config-crd.yaml

```diff
@@ -0,0 +1,298 @@
+                  properties:
+                    name:
+                      description: "name of the pullSecret secret"
+                      type: string
+                releaseImageUrl:
+                  description: |
+                    releaseImageUrl is the url to the image containing metadata for all
+                    components that run in the cluster, and is the primary and best way
+                    to specify what specific version of OpenShift you wish to install.
+                  type: string
+                sshPublicKey:
+                  description: sshPublicKey will be added to all cluster hosts for use in debugging.
+                  type: string
+                sshPrivateKeySecretRef:
+                  description: |
+                    sshPrivateKey is the private SSH key to use for access to
+                    compute instances. This private key should correspond to
+                    the public key included in the sshPublicKey. The
+                    private key is used by Hive to gather logs on the target
+                    cluster if there are install failures. The SSH private
+                    key is expected to be in the secret data under the
+                    ssh-privatekey key.
+                  type: object
+                  properties:
+                    name:
+                      description: "name of the privateKey secret"
+                      type: string
+                clusters:
+                  description: List of clusters tha will be installed in the site.
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      clusterName:
+                        description: |
+                          ClusterName is the friendly name of the cluster. It is used
+                          for subdomains, some resource tagging, and other instances
+                          where a friendly name for the cluster is useful.
+                        type: string
+                      baseDomain:
+                        description: BaseDomain is the base domain to which the cluster should belong.
+                        type: string
+                      clusterProfile:
+                        description: |
+                          clusterProfile speicfy the day0 configuration that will get appllied on the cluster must be one
+                          of the following stander, sno, du-sno, du, cu, cu-du
+                        type: string
+                        default: stander
+                        enum:
+                          - stander
+                          - sno
+                          - du-sno
+                          - du
+                          - cu
+                          - cu-du
+                      clusterLables:
+                        description: clusterLables specify the the day2 configuration policies that will get applied on the cluster.
```

Comment:
Typo: `clusterLabels`

Reply:
done

Reply:
Missed an additional typo in the description field too :)